packages/@uppy/core/src/UIPlugin.js

```diff
@@ -30,12 +33,10 @@ function debounce (fn) {
  * Use this for plugins that need a user interface.
  *
  * For plugins without an user interface, see BasePlugin.
- *
- * @param {object} main Uppy core object
- * @param {object} object with plugin options
- * @returns {Array|string} files or success/fail message
```

Comment:
Why remove JSDoc?

Reply:
I don’t see what it references, but it’s clearly out of place. I can make this in a separate commit or PR if you prefer.

Reply:
Yeah, this one is incorrect, I just meant if it was better to fix and not remove.